third-party/CMakeLists.txt

```diff
@@ -105,8 +105,8 @@ set(LUAROCKS_SHA256 cae709111c5701235770047dfd7169f66b82ae1c7b9b79207f9df0afb722
 set(UNIBILIUM_URL https://github.com/mauke/unibilium/archive/v1.2.0.tar.gz)
 set(UNIBILIUM_SHA256 623af1099515e673abfd3cae5f2fa808a09ca55dda1c65a7b5c9424eb304ead8)
 
-set(LIBTERMKEY_URL http://www.leonerd.org.uk/code/libtermkey/libtermkey-0.18.tar.gz)
-set(LIBTERMKEY_SHA256 239746de41c845af52bb3c14055558f743292dd6c24ac26c2d6567a5a6093926)
+set(LIBTERMKEY_URL https://github.com/neovim/libtermkey/archive/c0eb4e4a05f49ad8fee0195c77f2c29d09cc36af.tar.gz)
+set(LIBTERMKEY_SHA256 a9ea7207cc77ae72854721ab63a1096e32ed7e5b3a764ddce736d339d3553abe)
```

Comment:
Using [unreleased version](https://github.com/neovim/libtermkey/compare/a9b61424aae9f7548162ff112393c5f706cf54f1%5E...c0eb4e4a05f49ad8fee0195c77f2c29d09cc36af) of libtermkey which adds support for `termkey_hook_terminfo_getstr()`.